src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.ts

```diff
@@ -24,13 +24,13 @@ export class SavedScenariosComponent implements OnInit {
     this planning area can also view all the scenarios within it.
   `;
 
+  highlightedId: string = '-1';
   scenarios: ScenarioRow[] = [];
   displayedColumns: string[] = [
-    'id',
-    'starred',
+    'name',
     'projectAreas',
     'acresTreated',
-    'estimatedCost',
+    'estimated_cost',
```

Comment:
nit here: Seems like we are using camelCase for the rest of the column names, we should follow that convention